packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -1334,3 +1334,25 @@ function VM-Clean-Up {
     Write-Host "[+] Clearing up free space. This may take a few minutes..." -ForegroundColor Green
     VM-Clear-FreeSpace
 }
+
+function VM-Add-To-Path {
+    param (
+        [string]$pathToAdd
+    )
+    # Get the current user's Path environment variable
+    $currentPath = [System.Environment]::GetEnvironmentVariable("Path", [System.EnvironmentVariableTarget]::Machine)
+
+    # Check if the path already exists in the variable
+    if ($currentPath -notlike "*$pathToAdd*") {
```

Comment:
I think this doesn't work for all cases. In the following example `C:\ProgramData` is not in path, but a subdirectory:

![Screenshot 2023-12-21 at 12 20 06](https://github.com/mandiant/VM-Packages/assets/16052290/fd7a2223-d78f-40f9-891c-1bb0d6379db0)

Reply:
Ahh, that is true. I didn't think about adding a subdirectory of something already in the Path.

I'll write a fix! :)